src/Engine/Scene/Camera.cpp

```
@@ -14,7 +14,7 @@ Camera::Camera(glm::dvec3 position, glm::quat orientation):
 
 
 	m_config = {
-		.movementSpeed = 1e6f,
+		.movementSpeed = PhysicsConst::C,
```

Cppcheck (2.20.0)

[style] The function 'RegisterService' is never used.

(unusedFunction)